server.php

```diff
@@ -125,33 +125,32 @@ function valet_default_site_path($config)
  * Determine the fully qualified path to the site.
  * Inspects registered path directories, case-sensitive.
  */
-$valetSitePath = null;
-$domain = array_slice(explode('.', $siteName), -1)[0];
-
-foreach ($valetConfig['paths'] as $path) {
-    if ($handle = opendir($path)) {
-        while (false !== ($file = readdir($handle))) {
-            if (! is_dir($path.'/'.$file)) continue;
-            if (in_array($file, ['.', '..', '.DS_Store'])) continue;
-
-            // match dir for lowercase, because Nginx only tells us lowercase names
-            if (strtolower($file) === $siteName) {
-                $valetSitePath = $path.'/'.$file;
-                break;
+function get_valet_site_path($valtetConfig, $domain, $siteName)
```

Comment:
`$valtetConfig` seems to bemisspelled, and doesn't match the var in the rest of the function.

Reply:
Oh wow, what a mess, sorry. I guess I rushed it a bit too much there.